package/MDAnalysis/coordinates/DCD.py

```diff
@@ -216,7 +216,7 @@ def Writer(self, filename, n_atoms=None, **kwargs):
     def _frame_to_ts(self, frame, ts):
         """convert a dcd-frame to a :class:`TimeStep`"""
         ts.frame = self._frame
-        ts.time = (ts.frame + self._file.header['istart']) * self.ts.dt
+        ts.time = (ts.frame + self._file.header['istart']/self._file.header['nsavc']) * self.ts.dt
```

Comment:
I thought there where more places where we need changes. I'm pretty sure the writer needs to be updated too.

Reply:
I am pretty sure that you're right... I just committed what I had before falling asleep.

Reply:
I think I fixed all occurences of `istart`/`nsavc` in the reader and writer. I also added a test for the writer to check the defaults that we are setting. Namely, `istart=0` is not what CHARMM would do: it would set it to `nsavc` which is what MDAnalysis now does for `istart=None`. However, I kept the MDAnalysis value (which we had for a while) as the default.